python/lsst/meas/modelfit/display/cModelDisplay.py

```diff
@@ -0,0 +1,264 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+import lsst.afw.image as afwImage
+import lsst.meas.modelfit as measMod
+import lsst.shapelet as shapelet
+
+
+def displayReconstruedCmodel(exposure, record, cModelConfig, display="mpl"):
+    """ Display an image, the Cmodel, and residuals
+
+    Parameters
+    ----------
+    exposure : `lsst.afw.image.Exposure`
+        Exposure object that contains the source which was  modeled
+    record : `lsst.afw.table.SourceRecord`
+        Record object which contains the measurements made on the source
+    cModelConfig : `lsst.meas.modelfit.CModel(SingleFrame/Forced)Config`
+        Configuration object of the CModel plugin used in the measurement
+        process
+    display : `str`, optional
+        Display in which to render the data, may be mpl for matplotlib or afw
+        for afwDisplay, defaults to mpl
+
+    Raises
+    ------
+    `NotImplementedError`
+        If the display backend specified is not implemented
+    """
+
+    if display is "mpl":
+        displayReconstruedCmodel_mpl(exposure, record, cModelConfig)
+    elif display is "afw":
+        raise NotImplementedError("The afw display backend is not yet "
+                                  "implemented")
+    else:
+        raise NotImplementedError("The display backedn '{}' is not a supported"
+                                  "backend".format(display))
```

Comment:
Because at some point ds9 will be implemented, so I wanted to put the framework in for it now. This will catch any other string that users throw at it at that point.